### DISCUSSION TITLE
Checkbox styles not applying

### DISCUSSION THREAD

**Discussion by elenamarinaki** [govuk-react/govuk-react#1624] [Q&A]
Hello!

I would like some help with the following please: when using the `Checkbox` component, I cannot see the input element in my application at all, only the label. Although, when hovering over the area where it should be I can see the cursor changing to hand. The element seems to be here as when bringing back its opacity to 1, I can see the original checkbox. 

When running the example application in this repo, I can see all checkboxes just fine though.

Wondering what might be the issue, as I have followed the recommended settings but not sure if I need something else in my setup to make it work.
Any help would be greatly appreciated, thank you!

**Comment by user24** [comment]
I'm also seeing this issue. It's related to #1423.

As a workaround, add this to your project's package.json:

```
,
  "overrides": {
    "govuk-react": {
      "styled-components": "^5.3.0"
    }
  }
```

**Reply by elenamarinaki** [reply]
Thank you for this, will try this out :)
